scripts/rename_session_windows.py

```diff
@@ -236,8 +237,9 @@ def rename_windows(server: libtmux.Server):
             if not enabled_in_window:
                 continue
 
+            display_path = substitute_name(str(display_path), options.dir_substitute_sets)
```

Comment:
> I like the idea but does it work? It looks like you don't use `dir_substitute_sets` when renaming the window.

This line here uses `dir_substitute_sets` to substitute in the path before concatenating with the program name